pkg/cloud/services/eks/nodegroup.go

```diff
@@ -591,12 +590,7 @@ func (s *NodegroupService) setStatus(ng *eks.Nodegroup) error {
 		for _, group := range groups.AutoScalingGroups {
 			replicas += int32(len(group.Instances))
 			for _, instance := range group.Instances {
-				id, err := noderefutil.NewProviderID(fmt.Sprintf("aws://%s/%s", *instance.AvailabilityZone, *instance.InstanceId))
-				if err != nil {
-					s.Error(err, "couldn't create provider ID for instance", "id", *instance.InstanceId)
-					continue
-				}
-				providerIDList = append(providerIDList, id.String())
+				providerIDList = append(providerIDList, fmt.Sprintf("aws:///%s/%s", *instance.AvailabilityZone, *instance.InstanceId))
```

Comment:
I've been thinking about this over the weekend (including the fix from the end of last week) and the `///` doesn't feel right to me. I'm thinking that we need to change to `aws://%s/%s`

Reply:
https://github.com/kubernetes/cloud-provider-aws/blob/v1.25.0/pkg/providers/v1/instances.go#L53

CCM sets it like this.

Reply:
cc @yastij

Reply:
How about deciding this after merging this PR as all tests passed here? We can do a beta cut after this is merged and continue improving?

Reply:
Yes lets decide afterwards. 

I think some providers used the triple slash as they didn't naturally have separate segements...maybe?